src/pages/GovPages/Proposals/ProposalCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useContext, useState } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import { Clock } from 'react-feather'
+import { Flex } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import Row, { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import Card, { LightCard } from '../../../components/Card'
+
+import useInterval from '../../../hooks/useInterval'
+import HourGlass from '../../../assets/svg/hourglass.svg'
+import Block from '../../../assets/svg/block.svg'
+import Done from '../../../assets/svg/done_all.svg'
+
+const Container = styled(LightCard)<{ isPassed: number }>`
+  // 0 - voting in progress
+  // 1 - passed
+  // 2 - failed
+  padding: 1.25rem;
+  height: 88px;
+  position: relative;
+  cursor: pointer;
+  background: ${({ isPassed, theme }) =>
+    isPassed === 0
+      ? `linear-gradient(113.18deg, rgba(255, 255, 255, 0.35) -0.1%, rgba(0, 0, 0, 0) 98.9%), ${theme.dark1}`
+      : `linear-gradient(141.72deg, ${theme.bg1} -11.46%, ${
+          isPassed === 1 ? 'rgba(5, 122, 85, 0.5)' : 'rgba(238, 46, 81, 0.5)'
+        } 180.17%)`};
+  background-blend-mode: overlay, normal;
+  border: none;
+
+  ::before {
+    content: '';
+    background-image: linear-gradient(180deg, #14131d 0%, rgb(68 65 99 / 50%) 100%);
+    top: -1px;
+    left: -1px;
+    bottom: -1px;
+    right: -1px;
+    position: absolute;
+    z-index: -1;
+    border-radius: 8px;
+  }
+`
+
+const InfoText = styled(TYPE.main)<{ isPassed: number; small?: boolean }>`
+  font-style: normal;
+  font-weight: bold !important;
+  font-size: ${({ small }) => (small ? '9.5px !important' : '10px')};
+  line-height: 12px;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: ${({ theme, isPassed }) => (isPassed === 0 ? theme.purple3 : isPassed === 1 ? theme.green2 : theme.red1)};
+`
+
+const TextCard = styled(Card)<{ isPassed: number }>`
+  border: 1px solid
+    ${({ theme, isPassed }) => (isPassed === 0 ? theme.purple3 : isPassed === 1 ? theme.green2 : theme.red1)};
+  border-radius: 4px;
+  height: 16px;
+  padding: 2px 6px;
+  align-items: center;
+  width: auto;
+`
+
+const Progress = styled.div<{ isFor: boolean; width: string }>`
+  height: 2px;
+  margin: 0;
+  padding: 0;
+  width: ${({ width }) => width};
+  background-color: ${({ isFor, theme }) => (isFor ? theme.green2 : theme.red1)};
+`
+
+interface ProposalCardProps {
+  id: number
+  title: string
+  totalVote: number
+  for: number // percentage value
+  against: number // percentage value
+  isEnded: boolean
```

Comment:
Just a thought: I come from a Java background, and in Java (and other OOP languages), the `is` prefix is used exclusively on getters for boolean properties, while it is dropped for the property itself. Generally speaking, I think (personally) that not using an `is` as a prefix looks a bit better (even better in TS where you immediately know if you are using a bool or not, and you don't have doubts unlike with JS)